arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx

```diff
@@ -43,7 +43,11 @@ function sayHi() external view returns(string memory);
 
 Next, build Nitro by following steps 3-7 of the instructions in [How to build Nitro locally](/node-running/how-tos/build-nitro-locally). Note that if you've already built the Docker image, you still need run the last step to rebuild.
 
-Run Nitro by following the instructions in [How to run a full node](/node-running/how-tos/running-a-full-node#putting-it-all-together).
+Run Nitro by following the instructions (Note the instructions in [How to run a full node](/node-running/how-tos/running-a-full-node#putting-it-all-together) cann't start orbit node):
```

Comment:
```suggestion
Run Nitro with the following command:
```

Proposed change. It feels weird to link to a page when we're saying that you can't use the information in that page. Maybe we could add a link below the command with something like "for additional, optional parameters see [How to run a full node](/node-running/how-tos/running-a-full-node#optional-parameters)".
What do you think?